README.md

```diff
@@ -1,2 +1,85 @@
-# js-sdk
+<h1><p align="center"><img style="background: white;" alt="Rarimo JS SDK" src="https://scan.testnet.rarimo.com/android-chrome-512x512.png" width="256" /></p></h1>
+
+<div align="center">
+  <a href="https://github.com/rarimo/js-sdk/blob/main/LICENSE">
+    <img alt="License: MIT" src="https://img.shields.io/github/license/rarimo/js-sdk.svg" />
+  </a>
+</div>
+
+# Rarimo JavaScript SDK
 TypeScript-based source development script for Rarimo Protocol
+
+## Changelog
+Change log can be found by the following link: [CHANGELOG.md]
+
+[CHANGELOG.md]: https://github.com/rarimo/js-sdk/blob/main/CHANGELOG.md
+
+
+## API documentation
+
+The full API documentation is hosted at [rarimo.github.io/js-sdk]. This is a bit
+tricky to navigate and requires basic TypeScript understanding. It is helpful if
```

Comment:
`This is a bit  tricky to navigate and requires basic TypeScript understanding.` - don't undermine your own stuff heh. Just say it's TypeDoc